projects/plugins/jetpack/class.jetpack-autoupdate.php

```diff
@@ -344,12 +344,12 @@ public static function get_possible_failures() {
 	public static function get_plugin_slug( $plugin_file ) {
 		$update_plugins = get_site_transient( 'update_plugins' );
 		if ( isset( $update_plugins->no_update ) ) {
-			if ( isset( $update_plugins->no_update[ $plugin_file ] ) ) {
+			if ( isset( $update_plugins->no_update[ $plugin_file ] ) && isset( $update_plugins->no_update[ $plugin_file ]->slug ) ) {
```

Comment:
`isset` is smart enough that we can just do this
```suggestion
			if ( iisset( $update_plugins->no_update[ $plugin_file ]->slug ) ) {
```